tests/admin/controller/test_controller.py

```diff
@@ -6463,11 +6339,7 @@ def test_collection_library_registrations_post(self):
         ])
 
         class Mock(Registration):
-            # We reproduce the signature, even though it's not
-            # necessary for what we're testing, so that if the push()
-            # signature changes this test will fail.
-            def push(self, stage, url_for, catalog_url=None, do_get=None,
-                     do_post=None):
+            def push(self, *args, **kwargs):
```

Comment:
The version of this code that you remove here is the newer, more correct version.